common/minicurl.cc

```diff
@@ -21,18 +21,34 @@
  */
 
 #include "minicurl.hh"
+#include <time.h>
+#include <chrono>
 #include <curl/curl.h>
 #include <stdarg.h>
 #include <string.h>
 #include <sstream>
+#include "wforce_exception.hh"
 #include "dolog.hh"
 
+bool MiniCurl::initCurlGlobal()
+{
+  // This is guaranteed to be called only once
```

Comment:
how is it exactly guaranteed to called only once?

Reply:
The comment refers to the initialisation of the static below

Reply:
I can easily imagine it getting called more than once. If I am wrong, you could update the comment to explain why.

Reply:
Well you're right in that the initCurlGlobal() function will get called many times. But the curl_global_init call will only happen once. I'll clarify that in the comment

Reply:
I still don't get it. =)

Reply:
I need curl_global_init() to called only once. Doing it as part of the initialisation of a static variable is a way to achieve that, since statics are only initialized once.